analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml

```diff
@@ -55,6 +55,7 @@ analyzer:
     tool_versions: {}
   config:
     allow_dynamic_versions: false
+    skip_excludes: false
```

Comment:
Sorry for not noticing this earlier, but it just occurred to me that this should be called "skip_exclude*d*" (with "d" instead of "s") to align with the existing `--skip-excluded` CLI options, and to make it sound less like the excludes themselves would be skipped.

Reply:
Renamed.